### PR TITLE
[zh-cn] sync resource-quotas.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/policy/resource-quotas.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -362,17 +362,17 @@ The following types are supported:
...
+| `persistentvolumeclaims` | The total number of [PersistentVolumeClaims](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) that can exist in the namespace.                   |
+| `pods` | The total number of Pods in a non-terminal state that can exist in the namespace.  A pod is in a terminal state if `.status.phase in (Failed, Succeeded)` is true. |
+| `replicationcontrollers` | The total number of ReplicationControllers that can exist in the namespace.                                                                                        |
+| `resourcequotas` | The total number of ResourceQuotas that can exist in the namespace.                                                                                                |
+| `services` | The total number of Services that can exist in the namespace.                                                                                                      |
+| `services.loadbalancers` | The total number of Services of type `LoadBalancer` that can exist in the namespace.                                                                               |
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/policy/resource-quotas.md:373]: Seems these changes are problematic due to numerous redundancies.

Comment (tengqm) [reply content/zh-cn/docs/concepts/policy/resource-quotas.md:373]: These changes are unnecessary.

**Pull request merged**